src/ui/ui_render/ui_render_mixin.js

```diff
@@ -30,6 +31,38 @@ export function uiRenderMixin(kbnServer, server, config) {
   // render all views from ./views
   server.setupViews(resolve(__dirname, 'views'));
 
+  server.route({
+    path: '/bundles/app/{id}/bootstrap.js',
+    method: 'GET',
+    async handler(request, reply) {
+      const { id } = request.params;
```

Comment:
If we put everything within the `try {} catch {}`, and threw the error on 41 rather than passing it directly to `reply()`, then this handler would have a single flow for error and would properly route errors if `server.getUiAppById()` threw, or `request.params` wasn't defined for some reason.

Reply:
Good call